### PR TITLE
Feat(e2e):ONEPLAT-1079-Write test cases for User Groups.

### PR DESCRIPTION
**Pull request by supriyagawade**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

Added e2e tests for User Groups

## Does this PR introduce a breaking change
No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
hi @supriyagawade i noticed you putting `Cypress.config( 'includeShadowDom', true );` in almost every test spec files. I guess adding `includeShadowDom` to [cypress.json](https://github.com/1-Platform/one-platform/blob/master/cypress.json) would be a better option to do this rather than enabling it individually for reach test specs.

File: e2e/integration/usergrouptests.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    it( 'Test for header section', () => {
+        cy.get( '.app' ).within( () => {
+            cy.get( '.opc-header' ).within( () => {
+                cy.get( '.opc-header__top-row--header-name' ).should( 'contain.text', 'User Group SPA' )
+                cy.get( '.opc-header__bottom-row' ).should( 'be.visible' )
+                })
+            })
+        } );
+
+    /*it( 'Test for edit the group', () => {
```
Comment (ghanlohar) [inline e2e/integration/usergrouptests.js:23]: Remove commented code.

**Review by ghanlohar — CHANGES_REQUESTED** [review]

**Review by ghanlohar — COMMENTED** [review]
@supriyagawade Can you also check the file names for consistency.

**Review by riginoommen — APPROVED** [review]

**Review by gautamkrishnar — APPROVED** [review]
Apart from @ghanlohar's comments, rest LGTM.

**Review by ghanlohar — APPROVED** [review]
Looks good to me.

**Pull request merged**